doc/nrf/libraries/others/emds.rst

```diff
@@ -94,6 +94,40 @@ EMDS can store data within a guaranteed time, based on the amount of data being
 EMDS can be used to store data in memory in situations of critical power shortage, for example before the device battery is depleted.
 It is important that the hardware has the appropriate functionality to sustain power long enough for the storage to be completed before the power source is fully discharged.
 
+The easiest way of computing an estimate of the time required to store all entries, in a worst case scenario, is to call the :c:func:`emds_store_time_get` function.
+This function returns a worst-case storage time estimate in microseconds (µs) for a given application.
+For this to work, Kconfig options :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US`, :kconfig:option:`CONFIG_EMDS_FLASH_TIME_ENTRY_OVERHEAD_US` and :kconfig:option:`CONFIG_EMDS_FLASH_TIME_WRITE_ONE_WORD_US` need to be set as described in the `Implementation`_ section.
+The :c:func:`emds_store_time_get` function estimates the required worst-case time to store :math:`n` entries using the following formula:
+
+.. math::
+
+   t_\text{store} = t_\text{base} + \sum_{i = 1}^n \left(t_\text{entry} + t_\text{word}\left(\left\lceil\frac{s_\text{ate}}{s_\text{block}}\right\rceil + \left\lceil\frac{s_i}{s_\text{block}}\right\rceil \right)\right)
+
+where :math:`t_\text{base}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US`, :math:`t_\text{entry}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_ENTRY_OVERHEAD_US` and :math:`t_\text{word}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_WRITE_ONE_WORD_US`.
```

Comment:
I would suggest to elaborate a bit more a need in `T base` (that it covers case when erase was started right before `emds_store()` call).

Reply:
That is covered in the `Implementation` section, which is where you learn how to set these KConfig settings. I didn't want to repeat the information.